spec/rspec/sidekiq/matchers/be_unique_spec.rb

```diff
@@ -55,6 +55,57 @@
     include_context 'a unique worker'
   end
 
+  context '.until' do
+    let(:module_constant) { "Sidekiq::Enterprise" }
+    let(:expiration) { :success }
+    let(:interval) { 3.hours }
+    let(:sidekiq_options) { { unique_for: interval, unique_until: expiration } }
+    let(:worker) do
+      options = sidekiq_options
+      Class.new do
+        include ::Sidekiq::Worker
+        sidekiq_options options
+        def perform; end
+      end
+    end
+
+    before { stub_const(module_constant, true) }
+
+    subject do
+      stub_const('MuhWorker', worker)
+      MuhWorker
+    end
+
+    it { should be_unique.for(interval).until(:success) }
```

Comment:
🎨 Can you add something akin to this and the other subject below as examples to the README? As well as specifying that `SidekiqUniqueJobs` is not yet supported for this matcher?